chrome/locale/en-US/zotero/zotero.properties

```diff
@@ -1250,6 +1275,27 @@ publications.buttons.choose-sharing = Sharing
 publications.buttons.choose-license = Choose a License
 publications.buttons.addToMyPublications = Add to My Publications
 
+publications.my_publications = My Publications
+
+publications.intro = Items you add to My Publications will be shown on your profile page on org. If you choose to include attached files, they will be made publicly available under the license you specify. Only add work you yourself have created, and only include files if you have the rights to distribute them and wish to do so.
```

Comment:
"zotero." got lost?

Reply:
I've reinstituted `zotero.` prefix for locale strings ported from `publications.dtd` (which `Zotero.getString` doesn't retrieve from). There is a slight drawback of this: previously prefixed strings would only appear in `.xul` file, `.js` files used non-prefixed ones, now both kinds are mixed in `publicationsDialog.jsx`.

Reply:
Oh, no, sorry — we don't need (and don't want to) to prefix the string keys with `zotero.`. I was talking about `will be shown on your profile page on org` in the text. Presumably just a find-and-replace gone wrong.

Reply:
ohh... I see it now, thanks, fixed!